docs/writing-fusing-functions.md

```diff
@@ -67,9 +67,58 @@ main = λxλt1λf1 (λtλf (x f t) f1 t1)
 main = λxλt1λf1 (λf (x f f1) t1))
 main = λxλt1λf1 (x t1 f1)
 ```
-Wow! Simply by replacing lambda arguments with the values applied to them, we were able to make `(not (not x))` not grow in size. This is what fusing means, and it's a really powerful tool to make programs faster.
+Wow! Simply by replacing lambda arguments with the values applied to them, we were able to make `(not (not x))` not grow in size. This is what fusing means, and it's a really powerful tool to make programs faster. 
```

Comment:
```suggestion
Wow! Simply by replacing lambda arguments with the values applied to them, we were able to make `(not (not x))` not grow in size. This is what fusing means, and it's a really powerful tool to make programs faster.
```

Reply:
I removed this instance of trailing whitespace and other trailing whitespace in 26f4e65